.github/workflows/ci.yml

```diff
@@ -35,3 +35,9 @@ jobs:
         env:
           TOXENV: ${{ matrix.toxenv }}
         run: tox
+
+      - name: Run coverage
+        if: matrix.python-version == '3.8' && matrix.toxenv == 'py38'
+        uses: codecov/codecov-action@v1
+        with:
+          flags: unittests
```

Comment:
Do we want https://github.com/openedx/DoneXBlock/blob/master/.github/workflows/ci.yml#L43?